README.md

```diff
@@ -149,25 +229,23 @@ Information provided:
 - Idle : % of time CPU was idle.  
 - Nice : % of time spent by CPU executing user level processes with a nice priority.  
 - Iowait: % of time spent by CPU waiting for an outstanding disk I/O.  
-- Soft : % of time spent by the CPU servicing software interrupts.
--	Steal : % of time spent in involuntary waiting by logical CPUs.  
+- Soft : % of time spent by the CPU servicing software interrupts. - Steal : % of time spent in involuntary waiting by logical CPUs.
```

Comment:
Is this supposed to be in the same line?

Reply:
Huh, it shows that it's on two lines for me locally. Will add an explicit newline anyway.